.gitignore

```diff
@@ -1,3 +1,9 @@
 # Godot 4+ specific ignores
 .godot/
-.export/
\ No newline at end of file
+.export/
+
```

Comment:
Working in vscode, so added some new ignore statements. I'm also using a venv for gdlint/gdformat